adserver/api/views.py

```diff
@@ -74,6 +74,8 @@ class AdDecisionView(GeoIpMixin, APIView):
             The number and order must correspond to the ``div_ids``.
         :<json string priorities: An optional ``|`` delimited string of priorities for different ad types.
             The number and order matter, applying to ``div_ids`` and ``ad_types``.
+        :<json array keywords: An optional ``|`` delimited string of keywords.
+        :<json array campaign_types: An optional ``|`` delimited string of campaign types.
```

Comment:
We should probably define these concepts somewhere (eg. what's a campaign type? Keyword?), if we haven't. I had a user ask what they are recently when I linked them to this.